features/svg-opacity.yml

```diff
@@ -0,0 +1,15 @@
+name: SVG opacity
+description: The SVG opacity properties and attributes control the transparency of parts of an SVG.
```

Comment:
- Should `propertyize` the property name

- "parts of an SVG" is a bit generic, especially as this feature has a few distinct aspects, eg global vs element scope, CSS vs SVG-specific

Reply:
In revisting this, I think the filter and gradient opacity belong in filter/gradient features, so I removed them here, and expanded the description. https://github.com/web-platform-dx/web-features/pull/1604/commits/39c1b73dc85faa9f0654d4d64bf3e82495397235